browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -722,6 +735,15 @@ void VerticalTabStripRegionView::SetState(State state) {
 
   resize_area_->SetEnabled(state == State::kExpanded);
 
+  if (state == State::kExpanded) {
+    if (!toggle_button_highlight_.has_value()) {
+      toggle_button_highlight_ =
+          header_view_->toggle_button()->AddAnchorHighlight();
+    }
+  } else {
+    toggle_button_highlight_.reset();
+  }
```

Comment:
Instead of adding another var(`toggle_button_highlight_`), 
how about calling `header_view_->toggle_button()->SetHighlighted(state == State::kExpanded);` directly here and ctor? It seems working well for me but curious this has another issue?

Reply:
Yup, much better. Unlike bubble, we don't have life cycle issue here, so didn't have point to RAII style.